openpype/tools/workfile_template_build/lib.py

```diff
@@ -0,0 +1,29 @@
+import traceback
+
+from openpype.widgets import message_window
+
+
+def open_template_ui(builder, main_window):
+    """Open template from `builder`
+
+    Asks user about overwriting current scene and feedsback exceptions.
+    """
+
+    result = message_window.message(
+        title="Opening template",
+        message="Caution! This will overwrite your current scene.\n"
+        "Do you want to continue?",
```

Comment:
```suggestion
        message="Caution! This will overwrite your current scene.\n"
        "Do you want to continue?",
```
This doesn't sound correct - this will not overwrite your current scene right? It will only lose you any unsaved changes because it does not save your current scene. or does it actually save it into the current scene?

Reply:
You are right. You'll "only" loose the unsaved changes I guess.

Reply:
```suggestion
        message="Caution! You will loose unsaved changes.\n"
        "Do you want to continue?",
```